app/src/main/java/org/systers/mentorship/view/fragments/EditProfileFragment.kt

```diff
@@ -82,11 +83,24 @@ class EditProfileFragment: DialogFragment() {
         val editProfileDialog = dialog as AlertDialog
 
         editProfileDialog.getButton(AlertDialog.BUTTON_POSITIVE).setOnClickListener {
-            // TODO: Add validation
-            if (currentUser != editProfileBinding.user) {
+            val errors = validateProfileInput(editProfileBinding.user?.name?.trim())
+
+            with(editProfileBinding.tiName) {
+                this.error = when(errors.firstOrNull()) {
+                    is EditProfileFragmentErrorStates.EmptyNameError ->
+                        getString(R.string.error_empty_name)
+                    is EditProfileFragmentErrorStates.NameTooShortError ->
+                        context.getString(R.string.error_name_too_short)
```

Comment:
How can you getString without calling from context in line 91, but you call it here? @haroldadmin

Reply:
I'm sorry, this probably happened during the extraction of the string resource. It succesfully builds because the fragment class itself also has a method called `getString`, and so it slipped past me. I'll fix this right now.